bindings/matrix-sdk-crypto-js/src/responses.rs

```diff
@@ -156,7 +156,9 @@ impl DecryptedRoomEvent {
     /// trusted.
     #[wasm_bindgen(getter, js_name = "senderDevice")]
     pub fn sender_device(&self) -> Option<identifiers::DeviceId> {
-        Some(identifiers::DeviceId::from(self.encryption_info.as_ref()?.sender_device.clone()))
+        Some(identifiers::DeviceId::from(
+            self.encryption_info.as_ref()?.sender_device.as_ref()?.clone(),
+        ))
```

Comment:
Unrelated, but this could use `.into` instead of `DeviceId::from` because the output type can be inferred from the function signature.